### PR TITLE
Create user detail and edit page for new user management.

### PR DESCRIPTION
**Pull request by linuspahl**
This PR implements a user detail and edit page for new user management.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: graylog2-web-interface/src/components/users/ShowConfigValue.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <Col xs={3}>
+      <TextLabel>{label}</TextLabel>
+    </Col>
+    <Col xs={9}>
+      {value ?? '-'}
+    </Col>
+  </Row>
+);
+
+export default ShowConfigValue;
```
Comment (linuspahl) [inline graylog2-web-interface/src/components/users/ShowConfigValue.jsx:23]: @kmerz I just realized I've implemented `ShowConfigValue` and `FormFieldRead`, which do basically the same. I will update this in a follow-up PR as well.

File: graylog2-web-interface/src/components/users/UserDetails/ProfileSection.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    sessionActive,
+  },
+}: Props) => (
+  <SectionComponent title="Profile">
+    <ShowConfigValue label="Username" value={username} />
+    <ShowConfigValue label="Full name" value={fullName} />
+    <ShowConfigValue label="E-Mail Address" value={email} />
+    <ShowConfigValue label="Client Address" value={clientAddress} />
+    <ShowConfigValue label="Last Activity" value={lastActivity} />
+    <ShowConfigValue label="Logged In" value={sessionActive ? 'yes' : 'no'} />
```
Comment (kmerz) [inline graylog2-web-interface/src/components/users/UserDetails/ProfileSection.jsx:29]: In a follow up PR we would want a :green_circle:  here :smile:

Comment (kmerz) [reply graylog2-web-interface/src/components/users/UserDetails/ProfileSection.jsx:29]: no even better:

:green_circle: Logged in
:red_circle: Not logged in

or smth like that

File: graylog2-web-interface/src/components/users/form/FormFieldRead.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import * as React from 'react';
+
+import FormRow from './FormRow';
+
+type Props = {
+  label: string,
+  value: string,
+};
+
+const FormFieldRead = ({ label, value }: Props) => (
```
Comment (kmerz) [inline graylog2-web-interface/src/components/users/form/FormFieldRead.jsx:11]: I would suggest to call it ReadOnlyFormField

File: graylog2-web-interface/src/theme/GlobalThemeStyles.jsx [hunk 4]
```diff
@@ -98,10 +98,6 @@ const GlobalThemeStyles = createGlobalStyle(({ additionalStyles, theme }) => css
     }
   }
 
-  label {
-    font-size: ${theme.fonts.size.large};
```
Comment (kmerz) [inline graylog2-web-interface/src/theme/GlobalThemeStyles.jsx:102]: Are you sure?

**Pull request merged**